best-practices/README.md

```diff
@@ -101,6 +101,10 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryGirl factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Name service classes after the action they perform
+(e.g. `CreatePurchase`, `UpdateCreditCard`).
+* Expose one public method on services that does the work
```

Comment:
> Should it be the only public method?

Preferably yes (not counting `initialize`), to follow the Command pattern.

> Should it be a class method? Or an instance method?

Instance method, since we prefer to program to objects. A class convenience method can be added as well, per preference.

> If it's an instance method, should the initializer accept arguments or state?

Yes, I believe the state should be passed to the initializer.

@bernerdschaefer I'll re-word to clarify some of these, if others agree.